src/c++/perf_analyzer/periodic_concurrency_manager.cc

```diff
@@ -52,45 +51,53 @@ PeriodicConcurrencyManager::MakeWorker(
   return worker;
 };
 
+void
+PeriodicConcurrencyManager::MaybeAddConcurrentRequests()
```

Comment:
Change name to AddInRangeConcurrentRequests

Reply:
I think that name is more confusing. What this piece of logic is doing is only adding concurrent requests if we still have more requests to send.

In my opinion, separating out an if clause into its own method that requires a name is just asking for naming issues.

It seems short enough to just stay where it was. The if condition is difficult to put into words that can fit in a reasonably-lengthed function name.

I removed this function and refactored the if condition to be more clear:

```cpp
    uint64_t num_requests_sent{steps_completed_ * concurrency_range_.step};
    if (num_requests_sent < concurrency_range_.end) {
      AddConcurrentRequests(concurrency_range_.step);
    }
```